apps/obs-web/src/views/DebrieferAssessment.vue

```diff
@@ -1,49 +1,46 @@
 <template>
-    <div>
+    <div v-if="assessment">
```

Comment:
In the case they don't select an evaluation period rather than showing a blank page we should have some messaging like please select observer and evaluation period.

Reply:
oh yeah, good idea. Just added.